src/label.rb

```diff
@@ -0,0 +1,47 @@
+require 'securerandom'
+
+class Label
+  attr_accessor :title, :color
+  attr_reader :id, :items
+
+  def initialize(title, color, id = SecureRandom.random_number(1_000))
+    @title = title
+    @color = color
+    @id = id
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
+  end
```

Comment:
when adding a label, should update the label property in the item

Reply:
@carlosmuerto check it !